src/views/open-api-docs-view/utils/index.ts

```diff
@@ -13,7 +13,6 @@ export {
 export { getRequestData } from './get-request-data'
 export { getResponseData } from './get-response-data'
 export { groupOperations } from './group-operations'
-export { parseAndValidateOpenApiSchema } from './parse-and-validate-schema'
```

Comment:
This utility function was duplicative of the one at the `lib` level. Even though we do plan to delete the entire `src/views/open-api-docs-view` directory once migration is complete, it felt right to clean this up now to avoid confusion later.